src/main/resources/schemas/nhinc/common/LoadTestDataManagement.xsd

```diff
@@ -0,0 +1,407 @@
+            <xsd:element name="phonenumberId" type="xsd:long"/>
+        </xsd:sequence>
+    </xsd:complexType>
+    <xsd:element name="GetPhonenumberRequestMessage" type="tns:GetPhonenumberRequestMessageType" />
+
+    <xsd:complexType name="SavePatientRequestMessageType" >
+        <xsd:sequence>
+            <xsd:element name="configAssertion" type="nccommon:ConfigAssertionType"/>
+            <xsd:element name="patient" type="tns:PatientType"/>
+        </xsd:sequence>
+    </xsd:complexType>
+    <xsd:element name="SavePatientRequestMessage" type="tns:SavePatientRequestMessageType" />
+
+    <xsd:complexType name="SavePersonnameRequestMessageType" >
+        <xsd:sequence>
+            <xsd:element name="configAssertion" type="nccommon:ConfigAssertionType"/>
+            <xsd:element name="personname" type="tns:PersonnameType"/>
+        </xsd:sequence>
+    </xsd:complexType>
+    <xsd:element name="SavePersonnameRequestMessage" type="tns:SavePersonnameRequestMessageType" />
+
+    <xsd:complexType name="SaveAddressRequestMessageType" >
+        <xsd:sequence>
+            <xsd:element name="configAssertion" type="nccommon:ConfigAssertionType"/>
+            <xsd:element name="address" type="tns:AddressType"/>
+        </xsd:sequence>
+    </xsd:complexType>
+    <xsd:element name="SaveAddressRequestMessage" type="tns:SaveAddressRequestMessageType" />
+
+    <xsd:complexType name="SaveIdentifierRequestMessageType" >
+        <xsd:sequence>
+            <xsd:element name="configAssertion" type="nccommon:ConfigAssertionType"/>
+            <xsd:element name="identifier" type="tns:IdentifierType"/>
+        </xsd:sequence>
+    </xsd:complexType>
+    <xsd:element name="SaveIdentifierRequestMessage" type="tns:SaveIdentifierRequestMessageType" />
+
+    <xsd:complexType name="SavePhonenumberRequestMessageType" >
+        <xsd:sequence>
+            <xsd:element name="configAssertion" type="nccommon:ConfigAssertionType"/>
+            <xsd:element name="phonenumber" type="tns:PhonenumberType"/>
+        </xsd:sequence>
+    </xsd:complexType>
+    <xsd:element name="SavePhonenumberRequestMessage" type="tns:SavePhonenumberRequestMessageType" />
+
+    <xsd:complexType name="ListAllDocumentRequestMessageType" >
+        <xsd:sequence>
+            <xsd:element name="configAssertion" type="nccommon:ConfigAssertionType"/>
+        </xsd:sequence>
+    </xsd:complexType>
+    <xsd:element name="ListAllDocumentRequestMessage" type="tns:ListAllDocumentRequestMessageType" />
+
+    <xsd:complexType name="ListAllEventCodeRequestMessageType" >
+        <xsd:sequence>
+            <xsd:element name="configAssertion" type="nccommon:ConfigAssertionType"/>
+            <xsd:element name="documentId" type="xsd:long"/>
+        </xsd:sequence>
+    </xsd:complexType>
+    <xsd:element name="ListAllEventCodeRequestMessage" type="tns:ListAllEventCodeRequestMessageType" />
+
+    <xsd:complexType name="DuplicateDocumentRequestMessageType" >
```

Comment:
Do we need DuplicateCosumentRequestMessageType? Can we use the SaveDocumentRequestMessageType?

Reply:
no the function are different and portType don't like duplicated messages